recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.1.7.1" %}
+{% set sha256 = "47699ba598570ec4eb2f852a2e7db268767a4d3b9042510259dd4492dfc7dee0" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    fn: py_gd-{{ version }}.tar.gz
+    url: https://github.com/NOAA-ORR-ERD/py_gd/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    skip: True  # [py3k or win]
+    number: 0
+    script:
+        - pip install ./
```

Comment:
no -- but if I had done that, I would have found my problem sooner!

and --no-deps is a good idea, yes.

but I like the / -- I"ll keep that :-)

Something very weird is going on on my machine, but let's see how this goes